README.md

```diff
@@ -58,50 +58,54 @@ ClearSky is not just for a CLEAN feel for users - it's also for developers!<br>
 
 # <a name="SPA"></a>Español
 # Te presentamos: ClearSky-Sunrise
-ClearSky es un software de servidor rapido para Minecraft: Pocket Edition limpio y con caracteristicas estables. Fue diseñado para servidores de producción.
+ClearSky es un software de servidor rapido para Minecraft: Pocket Edition limpio y con caracteristicas estables. Fue diseñado para servidores en masa.
 
 **Estado de la compilación deTravisCI** [![TravisCI](https://travis-ci.org/ClearSkyTeam/ClearSky.svg?branch=0.14.0)](https://travis-ci.org/ClearSkyTeam/ClearSky "TravisCI Build Status")
 **Estado de la compilación de Jenkins** [![Build Status](http://jenkins.clearskyteam.org/buildStatus/icon?job=ClearSky)](http://jenkins.clearskyteam.org/job/ClearSky/ "Jenkins Build Status")
 
 
- - Tomamos codigo de otros proyectos pero lq mayoria de las caracteristicas las reescribimos para ClearSky.
+ - Tomamos codigo de otros proyectos pero la mayoria de las caracteristicas las reescribimos para ClearSky.
  - Nuestro grupo de Telegram, para que hables con nosotros: [Telegram/ClearSky] (https://telegram.me/joinchat/AlErxAY3tx0MPBGYuGtpDA).
- - [Mira como ClearSky abre así de rapido con con 70 plugins y 20 mundos](http://wolvesfortress.de/ezgif-1446650535.gif)
- - Inofficial Test Server : Creative IP: WolvesFortress.de Port: **19134** *Often offline, but if online, latest version*
+ - [Mira como ClearSky abre así de rapido con 70 plugins y 20 mundos](http://wolvesfortress.de/ezgif-1446650535.gif)
+ - Servidor de pruebas no oficial : IP Creativo: WolvesFortress.de Puerto: **19134** *A menudo fuera de línea, pero cuando está en línea posee la última versión*
 
 ## Donde consigo una compilación?
  - Consiguelo aqui: [Jenkins CI](http://jenkins.clearskyteam.org/).
  - Consigue un tiempo de ejecución de PHP optimizado: [ClearSkyTeam/PHPbinary](https://github.com/ClearSkyTeam/PHPbinary).
- - Consigue una demo de multicraft aqui: [ClearSkyTeam/MulticraftConfig](https://github.com/ClearSkyTeam/MulticraftConfig).
+ - Consigue una prueba de multicraft aqui: [ClearSkyTeam/MulticraftConfig](https://github.com/ClearSkyTeam/MulticraftConfig).
 
-## [CRITICAL]: Remueve xdebug en el servidor de producción.
+## [CRITICO!]: Remueve xdebug en el servidor de producción.
```

Comment:
[CRITICO!] Remover xdebug en el servidor de Producción.

Remover since you are telling them How to do it, not telling them TO DO it